src/model.cpp

```diff
@@ -0,0 +1,31 @@
+#include "../include/model.hpp"
+#include "raylib.h"
+#include <iostream>
+
```

Comment:
You can put the namespace indie one single time instaed of putting it before every method name

Reply:
Actually i can't because we using namespace to put a difference between Model of raylib and our Class 
even If i use "using namespace indie; " we can't compiling due to the ambiguity between instance of Model

Reply:
Done in last commit